api/utils/keys/yubikey.go

```diff
@@ -56,6 +60,8 @@ var (
 // matching the given touch requirement. This private key will either be newly generated
 // or previously generated by a Teleport client and reused.
 func getOrGenerateYubiKeyPrivateKey(touchRequired bool) (*PrivateKey, error) {
+	ctx := context.TODO()
```

Comment:
Please don't use context.TODO() unless you are planning a follow up.

In this case, could just add a `ctx` param to the method?

Reply:
This is followed up by https://github.com/gravitational/teleport/pull/31732

Reply:
I would add a comment like `// context.TODO removed by ##31732`, but feel free to ignore it.